draft-ietf-httpbis-client-cert-field.md

```diff
@@ -121,35 +121,39 @@ Finished messages.
 
 ## Encoding
 
-The field-values of the HTTP header field defined herein utilize the following
-encoded form.
+The header in this document encodes certificates as Structured Field Byte
+Sequences ({{Section 3.3.5 of RFC8941}}).
 
-A certificate is represented in text as an `EncodedCertificate`, which is the
-base64-encoded (Section 4 of {{!RFC4648}}) DER {{!ITU.X690.1994}} PKIX certificate. The
+The binary sequence is always a DER {{!ITU.X690.1994}} PKIX certificate. The
 encoded value MUST NOT include any line breaks, whitespace, or other additional
-characters. ABNF {{?RFC5234}} syntax for `EncodedCertificate` is shown in the
-figure below.
+characters. A binary sequence which cannot be successfully parsed as a
+certificate MUST be ignored.
```

Comment:
Is `MUST be ignored` too prescriptive here? I'd think some/many implementations would want to reject such a request. Maybe? And a MUST ignore would seem to preclude that.

Reply:
I think it MUST be rejected. What the system using the header parser does with that rejection is up to the security considerations, including ignoring a bad cert or rejecting the request outright. But from this draft's perspective, it's a rejection of a bad value.